lib/krikri/harvesters/primo_harvester.rb

```diff
@@ -0,0 +1,106 @@
+module Krikri::Harvesters
+  ##
+  # A harvester implementation for Primo
+  #
+  # Accepts options passed as `:primo => opts`
+  #
+  # Options allowed are:
+  #
+  #   - bulk_size: The number of records to fetch from Primo per request
```

Comment:
We'll need to have it accept additional Primo-specific options, which will vary across MWDL and Getty: `institution`, `loc`, `query`, and `query_exc` (see [the ingest details spreadsheet](https://docs.google.com/spreadsheets/d/1pACcWaevDNht7XbE_Sjs-Nnl4XWoQ2qqAeFaiBAxFMw/edit#gid=1534136574)). The first three are just string parameters, but `query_exc` should be able to take an array, which would lead to `query_exc` being added more than once to the request.